src/app_service/service/community/dto/community.nim

```diff
@@ -90,6 +90,8 @@ type CommunityDto* = object
   requestedToJoinAt*: int64
   isMember*: bool
   muted*: bool
+  isAlsoCuratedCommunity*: bool
+  available*: bool
```

Comment:
I think we do not need to introduce this property. We could [deduce](https://github.com/status-im/status-go/blob/f60716412259aece88d899d09fc71de4c4ba5d2e/protocol/messenger_communities.go#L1631) if the community is available:

```nim
proc isAvailable*(communityDto: CommunityDto): bool =
  return communityDto.name != "" and communityDto.description != ""
```

Otherwise `available` will be very misleading for communities other than those from the curated directory.

Reply:
Good idea @osmaczko . Updated